composer.json

```diff
@@ -1,28 +1,38 @@
 {
   "name": "jangregor/phpstan-prophecy",
-  "type": "phpstan-extension",
   "description": "Provides a phpstan/phpstan extension for phpspec/prophecy",
   "license": "MIT",
+  "type": "phpstan-extension",
   "authors": [
     {
       "name": "Jan Gregor Emge-Triebel",
       "email": "jan@jangregor.me"
     }
   ],
   "require": {
-    "php": "^7.1 || ^8.0",
+    "php": "^7.2 || ^8.0",
     "phpstan/phpstan": "^1.0.0"
   },
-  "conflict": {
-    "phpspec/prophecy": "<1.7.0,>=2.0.0",
-    "phpunit/phpunit": "<6.0.0,>=10.0.0"
-  },
   "require-dev": {
     "ergebnis/composer-normalize": "^2.1.1",
     "ergebnis/license": "^1.0.0",
     "ergebnis/php-cs-fixer-config": "~2.2.0",
-    "phpspec/prophecy": "^1.7.0",
-    "phpunit/phpunit": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^9.0.0"
+    "phpspec/prophecy": "^1.17.0",
+    "phpunit/phpunit": "^8.0.0 || ^9.0.0"
```

Comment:
prophecy 1.17 does only support PHPUnit 8, 9 (10 soon). So we can not longer test against 6 and 7.